data/contrib_data.py

```diff
@@ -32,6 +32,9 @@ def import_data(contributor):
     try:
         contributor['issues_opened'] = contributor.pop('issues')
         contributor['num_commits'] = contributor.pop('contributions')
+        contributor_location = contributor.get('location')
+        if contributor_location:
+            contributor['location'] = contributor_location
```

Comment:
You can directly do `contributor['location'] = contributor.get('location')`

If there is no `location` field you will have `None`. And `location` field can accept null values. https://github.com/coala/community/pull/250/files#diff-6556cd068e37d3b094ccb8363a7e3a7dR18